todo_app/data/trello_items.py

```diff
@@ -0,0 +1,70 @@
+# create a class
+
+import requests
+import dotenv
+import os
+import json
+
+dotenv.load_dotenv("../.env")
```

Comment:
This should be unnecessary since Flask automatically tries to pick this file up when it runs; I was able to run your app both in & out of debug without issues after removing this line